libs/sx1302_hal/patches/000-edit_cflags.patch

```diff
@@ -0,0 +1,22 @@
+--- a/libloragw/Makefile
++++ b/libloragw/Makefile
```

Comment:
Missing Git header, which contains usual information like: who is the author, the commit subject, commit description and from that, we can see if this is a hack or it could be upstreamed.